custom_components/myskoda/binary_sensor.py

```diff
@@ -222,13 +222,13 @@ def is_supported(self) -> bool:
         return False
 
 
-class LightsOn(StatusBinarySensor):
-    """Detects whether the lights are on."""
+class ParkingLightsOn(StatusBinarySensor):
```

Comment:
I guess we'll need to call this out in the release notes so people know the `lights_on` sensor is gone.

I imagine HA has build in mechanisms to 'migrate' things but at this stage in the development I'm ok to just ask users to handle it 😅

Reply:
SemVer tells us this is 2.0.0 then :open_mouth:

Reply:
I could make it a bit less intrusive and only update icon and translation. That may be a better way forward.